src/main/scala/ai/privado/model/Constants.scala

```diff
@@ -162,5 +162,5 @@ object Constants {
   val semanticDelimeter = "_A_"
 
   // Ruby defaults
-  val defaultExpansionLimit = 20
+  val defaultExpansionLimit = 100
```

Comment:
What is impact of this change?

Reply:
Increase in this limit allows us to generate more tasks in dataflow operation, In dataflow engine the default limit is 1000